Tools/scripts/build_ci.sh

```diff
@@ -246,19 +246,19 @@ for t in $CI_BUILD_TARGET; do
 
     if [ "$t" == "stm32f7" ]; then
         echo "Building mRoX21-777/"
-        $waf configure --Werror --board mRoX21-777
```

Comment:
we need to keep --Werror on CI

Reply:
So, does it mean I should I try to come up with solution on libcanard side? I don't see what else can I fix on ardupilot side.

Reply:
Why are there warnings?

Reply:
Sorry for the long answer. If disabling --Werror is unacceptable and we can't do something on the ardupilot side, so it might be better to move this discussion to the libcanard side. I've implemented a simple solution and opened a pull request: https://github.com/OpenCyphal/libcanard/pull/197.

Reply:
fixed on the libcanard side